Doc/library/asyncio-policy.rst

```diff
@@ -112,6 +112,11 @@ asyncio ships with the following built-in policies:
 
       On Windows, :class:`ProactorEventLoop` is now used by default.
 
+   .. deprecated:: 3.11.1
+      :meth:`get_event_loop` now emits a :exc:`DeprecationWarning` if there
+      is no current event loop set and a new event loop has been implicitly
+      created. In Python 3.12 it will be an error.
+
```

Comment:
Note: it's weird to have this note here in 3.10 docs about a 3.11.1 deprecation but at the same time we *are* making this change too to 3.10.9. Should I say "3.10.9" here instead? Similarly, Serhiy used phrasing "undeprecated in 3.11.1" across tests. Should that read 3.10.9 in this backport?

Reply:
I changed everything to 3.10.9 for consistency.